### DISCUSSION TITLE
Is there a way we can select the default tab other than the first tab in tabbed forms ?

### DISCUSSION THREAD

**Discussion by venkateshpullaganti** [manojadams/mui-forms#64] [Q&A] [answered]
In the below form I want to select the second tab `Other Details` as a default tab. Is that possible ?
<img width="1512" alt="Screenshot 2024-04-16 at 1 29 45 PM" src="https://github.com/manojadams/mui-forms/assets/38013987/16cb107d-6a1a-4e20-8cf5-b879f5ee657a">

**Comment by manojadams** [comment]
@venkateshpullaganti @venkateshpullaganti 
You can use the property `pageNumber` to open a particular tab initiallty.
Note that page number starts from 1.

Here is a sample:
```
    <MuiForms
      className="s-form"
      schema={schema}
      pageNumber={2}
      onSubmit={(data) => {
          console.log(data);
      }}
  />
```

**Reply by venkateshpullaganti** [reply, marked as answer]
Thank you @manojadams 

I guess we need to work on the documentation part also.
